waiter/src/waiter/core.clj

```diff
@@ -864,8 +864,10 @@
                                      (f))))))))))})
 
 (def scheduler
-  {:scheduler (pc/fnk [[:settings scheduler-config scheduler-syncer-interval-secs]
+  {:scheduler-promise-chan (pc/fnk [] (async/promise-chan))
```

Comment:
Nitpick: comment about needing this to break the cycle in initialization.

Reply:
Added comment.